src/app/components/users/controllers/registration.js

```diff
@@ -0,0 +1,17 @@
+angular.module('librarium')
+  .controller('RegistrationController',
+    ['UsersService', '$state', function (usersService, $state) {
+      const self = this;
+      self.form = {};
+      const newUser = { user: self.form };
+
+      self.submitForm = function (isValid) {
+        if (isValid) {
+          usersService.addUser(newUser).then(function () {
+            $state.go('books');
```

Comment:
after logging the user you should save the user in local storage using the session service